css/fullcalendar.scss

```diff
@@ -128,10 +128,12 @@
 }
 
 .fc-day-header.fc-today {
-	background-color: var(--color-primary) !important;
-	color: var(--color-primary-text);
-	opacity: 1;
-	font-weight: bold !important;
+	> a, span {
+		background-color: var(--color-primary);
+		border-radius: var(--border-radius-pill);
+		padding: 2px 6px;
+		color: var(--color-primary-text);
```

Comment:
We should add `font-weight: bold;` here so it’s like in the month view. :)

Also, we need to add a general:
```
.fc .fc-today {
    opacity: 1;
}
```
So that the blue is the proper color and not slightly transparent (lighter) blue. :)

Reply:
Done 👍